pydmd/bopdmd.py

```diff
@@ -1461,6 +1469,108 @@ def fit(self, X, t):
 
         return self
 
+    def fit_econ(self, s, V, t):
+        """
+        Compute the Optimized Dynamic Mode Decomposition using the
+        SVD results of the snapshot matrix.
+        Use this method if you have pre-computed the SVD and the
+        original snapshot matrix is not avaialble or too large to
+        store in memory.
+        The left singular vectors (U) must be specified as the projection
+        basis when initializing the BOPDMD object.
+
+        :param s: the singular values.
+        :type s: numpy.ndarray
+        :param V: the right singular vectors.
+        :type V: numpy.ndarray
+        :param t: the input time vector.
+        :type t: numpy.ndarray
+        """
+        self._reset()
+        self._time = np.array(t).squeeze()
+
+        if self._proj_basis is None or not self._use_proj:
+            msg = """
+            proj_basis must be provided when using fit_econ,
+            and use_proj must be set to True.
+            """
+            raise ValueError(msg)
+        elif (
```

Comment:
`else` clause not needed here
```suggestion
        if (
```